pkg/yttlibrary/url.go

```diff
@@ -187,6 +189,22 @@ func (b urlModule) QueryParamsDecode(thread *starlark.Thread, f *starlark.Builti
 	return core.NewGoValue(result).AsStarlarkValue(), nil
 }
 
+// allowQuerySemicolons restores pre-Go 1.17 handling of query parameters.
+// Doing so defers making a breaking change as we upgrade from Go 1.16 to 1.17.
+// We expect to pass along this breaking change to end-users at some point, but not at this time (circa v0.37.x).
+// Shamelessly stolen from https://cs.opensource.google/go/go/+/refs/tags/go1.17.2:src/net/http/server.go;l=2892-2908
+// See also:
+// - https://golang.org/doc/go1.17#semicolons
+// - https://github.com/golang/go/issues/25192
+//   - particularly https://github.com/golang/go/issues/25192#issuecomment-789799446 which spells-out the vulnerability
+//     specifically.
+func (b urlModule) allowQuerySemicolons(encodedVal string) string {
+	if strings.Contains(encodedVal, ";") {
+		return strings.ReplaceAll(encodedVal, ";", "&")
```

Comment:
could just be strings.ReplaceAll(...) without an if?

Reply:
Yeah, I thought about that... and at one point I had included output to stderr that presented a warning (and then decided that might create more trouble than it was worth)... but forgot to remove the `if` when I walked that back.

Will do.

Reply:
done